backend/swagger.json

```diff
@@ -188,6 +192,147 @@
+                      "example": ["comment2","comment2"]
+                    },
+                    "date":{
+                      "type": "string",
+                      "description": "Start day of the event",
+                      "example": "11/12/2013"
+                    },
+                    "authorId":{
+                      "type": "string",
+                      "description": "Name of the author",
+                      "example": "Cumhur Kılıç"
+                    },
+                    "rank":{
+                      "type": "number",
+                      "description": "Rank of the event",
+                      "example": 4
+                    },
+                    "voterNumber":{
+                      "type": "number",
+                      "description": "Number of voter for event",
+                      "example": 156
+                    },
+                    "country":{
+                      "type": "string",
+                      "description": "Country of the event",
+                      "example": "Turkey"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "/event/{_id}": {
+        "get": {
+          "tags": [
+            "event"
+          ],
+          "summary": "Returns requested event with historical data",
+          "consumes": [
+            "application/json"
+          ],
+          "produces": [
+            "application/json"
+          ],
+          "parameters": [
+            {
+              "name": "_id",
+              "in": "path",
+              "description": "Unique event ID",
+              "required": true,
+              "type": "string"
+            }
+          ],
+  
+          "responses": {
+            "200":{
+              "description": "OK. Event returned succesfully.",
+              "schema": {
+                "type": "object",
+                "properties":  {
+                  "title":{
+                    "type": "string",
+                    "description": "Title of the event",
+                    "example": "title"
+                  },
+                  "body":{
+                    "type": "string",
+                    "description": "Event body",
+                    "example": "272.7 example"
+                  },
+                  "comment":{
+                    "type": "array",
+                    "description": "Comments of the event",
+                    "example": ["comment2","comment2"]
+                  },
+                  "date":{
+                    "type": "string",
+                    "description": "Start day of the event",
+                    "example": "11/12/2013"
+                  },
+                  "authorId":{
+                    "type": "string",
+                    "description": "Name of the author",
+                    "example": "Cumhur Kılıç"
+                  },
+                  "rank":{
```

Comment:
What is the range of this rank ? (investing.com uses 1 to 3)? If so, we should add this restriction to the model